source/extensions/filters/http/grpc_http1_reverse_bridge/filter.cc

```diff
@@ -203,6 +203,31 @@ Http::FilterDataStatus Filter::encodeData(Buffer::Instance& buffer, bool end_str
   }
 }
 
+Http::FilterTrailersStatus Filter::encodeTrailers(Http::HeaderMap& trailers) {
+  trailers.insertGrpcStatus().value(grpc_status_);
+
+  if (withhold_grpc_frames_) {
```

Comment:
This code is mostly copied. Please figure out a way to unify it into a common utility function.

Reply:
Done. Moved the duplicated code into `Filter::buildGrpcFrameHeader`